spec/javascripts/support/acuant/document_capture_dom.js

```diff
@@ -39,13 +39,23 @@ const INITIAL_HTML = `
 </p>
 `;
 
+let originalWindow;
+let originalDocument;
+
 export const setupDocumentCaptureTestDOM = () => {
+  originalWindow = global.window;
+  originalDocument = global.document;
```

Comment:
Small error here: Since `setupDocumentCaptureTestDOM` runs in `beforeEach` (before _each_ test case) and `teardownDocumentCaptureTestDOM` runs in `after` (after _all_ have finished), `originalWindow` assignment here might actually be assigning from a previous test case within the same Acuant test suite, thus overriding the true "original". 

This was encountered and fixed in #3952 (ff6eac0519a61956f12081f5fff19bc259db61e2).